tests/InputFiles/GeneralizedHarmonic/GaugeWave.yaml

```diff
@@ -16,21 +16,17 @@ Evolution:
       Order: 1
 
 DomainCreator:
-    Shell:
-      InnerRadius: 1.9
-      OuterRadius: 2.3
-      InitialRefinement: 0
-      InitialGridPoints: [5, 5]
-      UseEquiangularMap: true
-      AspectRatio: 1.0
-      UseLogarithmicMap: true
-      RadialBlockLayers: 1
+  Brick:
+    LowerBound: [-2.0, -2.0, -2.0]
+    UpperBound: [2.0, 2.0, 2.0]
+    InitialRefinement: [0, 0, 0]
+    InitialGridPoints: [5, 5, 5]
+    IsPeriodicIn: [true, true, true]
 
```

Comment:
In the last long, higher-resolution run I ran, I used
~~~

DomainCreator:
    Brick:
      LowerBound: [-0.5, -0.5, -0.5]
      UpperBound: [0.5, 0.5, 0.5]
      InitialRefinement: [0, 0, 0]
      InitialGridPoints: [5, 5, 5]
      IsPeriodicIn: [false, false, false]
AnalyticSolution:
  GaugeWave:
    Amplitude: 0.01
    Wavelength: 10.0
~~~